node/src/BaseClient.ts

```diff
@@ -1425,6 +1428,47 @@ export class BaseClient {
         return this.createWritePromise(createLLen(key));
     }
 
+    /**
+     * Atomically pops and removes the left/right-most element to the list stored at `source`
+     * depending on `where_from`, and pushes the element at the first/last element of the list
+     * stored at `destination` depending on `where_to`.
+     *
+     * See https://valkey.io/commands/lmove/ for details.
+     *
+     * @param source - The key to the source list.
+     * @param destination - The key to the destination list.
+     * @param where_from - The direction to remove the element from (`ListDirection.LEFT` or `ListDirection.RIGHT`).
+     * @param where_to - The direction to add the element to (`ListDirection.LEFT` or `ListDirection.RIGHT`).
```

Comment:
Use camelCase (`whereFrom`, `whereTo` - update everywhere) and use `{@link ...` for enum references.